core/server/common/src/main/java/alluxio/web/WebServer.java

```diff
@@ -91,7 +91,8 @@ public WebServer(String serviceName, InetSocketAddress address) {
     }
 
     System.setProperty("org.apache.jasper.compiler.disablejsr199", "false");
-    mServletContextHandler = new ServletContextHandler(ServletContextHandler.SECURITY);
+    mServletContextHandler = new ServletContextHandler(ServletContextHandler.SECURITY
+        | ServletContextHandler.NO_SECURITY);
```

Comment:
we want `NO_SESSIONS` instead of `NO_SECURITY`

Reply:
good catch. didn't notice.